Marlin/src/core/boards.h

```diff
@@ -428,6 +428,10 @@
 #define BOARD_FYSETC_SPIDER_KING407   4242  // FYSETC Spider King407 (STM32F407ZG)
 #define BOARD_MKS_SKIPR_V1            4243  // MKS SKIPR v1.0 all-in-one board (STM32F407VE)
 #define BOARD_TRONXY_V10              4244  // TRONXY V10 (STM32F446ZE)
+#define BOARD_ZNP_ROBIN_NANO_V1_3     4245  // Elegoo Neptune 2 v1.3 board
+#define BOARD_MKS_NEPTUNE_X           4246  // Elegoo Neptune X
+#define BOARD_ZNP_ROBIN_NANO          4247  // Elegoo Neptune 2 v1.2 board
```

Comment:
This is a clone of the MKS Robin Nano 1.2, so not sure if you want to include it.

Reply:
As above, it looks like the bootloader may be different, requiring the firmware binary to be named elegoo.bin for it to be seen on the SD card, but I will need to confirm.